packages/backend/src/index.ts

```diff
@@ -44,13 +44,7 @@ import {
   gatherDynamicPluginsSchemas,
 } from './schemas';
 
-// TODO(davidfestal): The following import is a temporary workaround for a bug
-// in the upstream @backstage/backend-plugin-manager package.
-//
-// It should be removed as soon as the upstream package is fixed and released.
-// see https://github.com/janus-idp/backstage-showcase/pull/600
 import { WinstonLogger } from '@backstage/backend-app-api';
```

Comment:
```suggestion
```
Remove this and move merge it with the imports from `@backstage/backend-app-api` at the top of the file. Can't seem to select it though in the review panel.